src/app/trainee/revalidation-table/revalidation-table.component.html

```diff
@@ -0,0 +1,85 @@
+<div class="table-container">
+    <table mat-table [dataSource]="recommendationsHistory" multiTemplateDataRows class="w-100">
+        <caption class="collapsed no-height">
+            Trainee revalidation historical details
+        </caption>
+        <ng-container matColumnDef="recommendation">
+            <th mat-header-cell scope="col" *matHeaderCellDef>Recommendation</th>
+            <td mat-cell *matCellDef="let element">
+                {{ element.recommendation }}
+            </td>
+        </ng-container>
+        <ng-container matColumnDef="outcome">
+            <th mat-header-cell scope="col" *matHeaderCellDef>Outcome</th>
+            <td mat-cell *matCellDef="let element">
+                <div class="outcome" [ngSwitch]="element.outcome">
+                    <mat-icon *ngSwitchCase="'Approved'" aria-hidden="true" class="icon-green">done_all
```

Comment:
would be good to use an enum or something for the Approved/Rejected/Under review status

Reply:
Absolutely. No enum or values decided yet for non-gmc related status. Will create enum in anticipation for real values as all these are mocked.

Reply:
done